### PR TITLE
Feature/4292 fix capitalisation on initial sync screen

### PR DESCRIPTION
**Pull request by AdamTomaja**
Simple change fixing some texts on initial sync screen
### Pull Request Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: vector/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -197,16 +197,16 @@
...
-    <string name="initial_sync_start_importing_account_left_rooms">Initial Sync:\nImporting Left Rooms</string>
-    <string name="initial_sync_start_importing_account_groups">Initial Sync:\nImporting Communities</string>
-    <string name="initial_sync_start_importing_account_data">Initial Sync:\nImporting Account Data</string>
+    <string name="initial_sync_start_server_computing">Initial sync:\nWaiting for server response…</string>
+    <string name="initial_sync_start_downloading">Initial sync:\nDownloading data…</string>
+    <string name="initial_sync_start_importing_account">Initial sync:\nImporting account…</string>
+    <string name="initial_sync_start_importing_account_crypto">Initial sync:\nImporting crypto</string>
+    <string name="initial_sync_start_importing_account_rooms">Initial sync:\nImporting rooms</string>
+    <string name="initial_sync_start_importing_account_joined_rooms">Initial sync:\nLoading your conversations\nIf you\'ve joined lots of rooms, this might take a while</string>
+    <string name="initial_sync_start_importing_account_invited_rooms">Initial sync:\nImporting invited Rooms</string>
```
Comment (bmarty) [inline vector/src/main/res/values/strings.xml:206]: `Rooms` -> `rooms`

File: vector/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -197,16 +197,16 @@
...
+    <string name="initial_sync_start_importing_account_left_rooms">Initial sync:\nImporting left Rooms</string>
```
Comment (bmarty) [inline vector/src/main/res/values/strings.xml:207]: `Rooms` -> `rooms`

File: vector/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -197,16 +197,16 @@
...
+    <string name="initial_sync_start_importing_account_groups">Initial sync:\nImporting communities</string>
+    <string name="initial_sync_start_importing_account_data">Initial sync:\nImporting account Data</string>
```
Comment (bmarty) [inline vector/src/main/res/values/strings.xml:209]: `Data` -> `data`

**Review by bmarty — COMMENTED** [review]
Thanks for your PR. Can you fix some remaining capitalisations please?

**Comment by ouchadam** [comment]
@bmarty how do we want to handle the other locales with similar issues, should we raise separate issues for them?

**Comment by AdamTomaja** [comment]
> Thanks for your PR. Can you fix some remaining capitalisations please?

done :)

**Review by bmarty — APPROVED** [review]
Thanks for the update!
Will be in release 1.3.7

**Pull request merged**